.github/workflows/perf_test.yml

```diff
@@ -0,0 +1,66 @@
+name: Performance Tests
+
+on:
+  pull_request:
```

Comment:
I think we should just do on pushes to staging right now. Maybe pushes to master but pushes to staging should include that as well. Probably don't want pull_request to master

Reply:
updated